java/spacewalk-java.spec

```diff
@@ -106,13 +106,12 @@ BuildRequires:  httpcomponents-client
 BuildRequires:  ical4j
 BuildRequires:  jade4j
 BuildRequires:  (gnu-jaf or jakarta-activation)
-%if 0%{?rhel} >= 8 || 0%{?fedora}
+%if 0%{?rhel} || 0%{?fedora}
```

Comment:
Why are we removing `>= 8`?

This means someone could think this can be built for CentOS7.

Reply:
@juliogonzalez EL7 is not supported so I removed it. This is server side only. I would see following sensible combinations (unless there are specific edge cases)
- `0%{?rhel}` - all supported versions.
- `0%{?rhel} == 8` - anything 8 specific.
- `0%{?rhel} >= 9` - anything 9 specific.

If we want to keep 7 in there then I happily revert. My thoughts are that the logic might get too clustered in future.